pkgs/applications/misc/rofi/default.nix

```diff
@@ -18,18 +19,22 @@
 , flex
 , librsvg
 , check
+, waylandSupport ? false, wayland-protocols, wayland
```

Comment:
iirc you need to format with nixpkgs-fmt, which will put these on separate lines

Reply:
Putting optional inputs depending on a boolean option next to it (or more generally, related options on the same line) by hand seems to be a common style (e.g. in [enlightenment](https://github.com/NixOS/nixpkgs/blob/43a211293497e1c2596836f8c711f3319dd667d3/pkgs/desktops/enlightenment/enlightenment/default.nix#L16), [waffle](https://github.com/NixOS/nixpkgs/blob/43a211293497e1c2596836f8c711f3319dd667d3/pkgs/development/libraries/waffle/default.nix#L14), [firefox](https://github.com/NixOS/nixpkgs/blob/43a211293497e1c2596836f8c711f3319dd667d3/pkgs/applications/networking/browsers/firefox/common.nix#L26), [bemenu](https://github.com/NixOS/nixpkgs/blob/43a211293497e1c2596836f8c711f3319dd667d3/pkgs/applications/misc/bemenu/default.nix#L4), [deadbeef](https://github.com/NixOS/nixpkgs/blob/43a211293497e1c2596836f8c711f3319dd667d3/pkgs/applications/audio/deadbeef/default.nix), [curl](https://github.com/NixOS/nixpkgs/blob/43a211293497e1c2596836f8c711f3319dd667d3/pkgs/tools/networking/curl/default.nix), [qemu](https://github.com/NixOS/nixpkgs/blob/43a211293497e1c2596836f8c711f3319dd667d3/pkgs/applications/virtualization/qemu/default.nix) to name a few).
I'm totally fine with full auto-formatting though :+1:

Reply:
Update: running `git clone --depth=1 git@github.com:NixOS/nixpkgs && nixpkgs-fmt nixpkgs` reformats about half of the files (11639 / 24067), so I guess auto-formatting is not a hard requirement :laughing:

Reply:
yeah, I think formatting is done file by file for new ones or similar, incrementally until all are changed ideally.
Try to only format this file

Reply:
I see, done!